app/internal/page/data.go

```diff
@@ -225,3 +226,46 @@ type Certificate struct {
 	AgreeToStatement bool
 	Agreed           time.Time
 }
+
+type AddressDetail struct {
+	Name    string
+	Role    actor.Type
+	Address place.Address
+	ID      string
+}
+
+func (l *Lpa) ActorAddresses() []AddressDetail {
+	var ads []AddressDetail
+
```

Comment:
and the donor's address?

Reply:
although maybe don't care too much, since this is more about putting something together to look at

Reply:
I'd originally left this out after seeing a screen of the same address between actors but this was before I'd added the actor type in the template. I can definitely see the value now so will add back in 👍